src/commands/protocols/common/simple-asset-sync-command.js

```diff
@@ -0,0 +1,177 @@
+        } = command.data;
+
+        await this.operationIdService.updateOperationIdStatus(
+            operationId,
+            blockchain,
+            OPERATION_ID_STATUS.COMMIT_PROOF.SIMPLE_ASSET_SYNC_START,
+        );
+
+        this.logger.info(
+            `[SIMPLE_ASSET_SYNC] (${operationId}): Started command for the ` +
+                `Blockchain: ${blockchain}, Contract: ${contract}, Token ID: ${tokenId}, ` +
+                `Keyword: ${keyword}, Hash function ID: ${hashFunctionId}, Epoch: ${epoch}, ` +
+                `State Index: ${stateIndex}, Retry number: ${
+                    COMMAND_RETRIES.SIMPLE_ASSET_SYNC - command.retries + 1
+                }`,
+        );
+
+        const ual = this.ualService.deriveUAL(blockchain, contract, tokenId);
+
+        const getOperationId = await this.operationIdService.generateOperationId(
+            OPERATION_ID_STATUS.GET.GET_START,
+        );
+
+        this.logger.debug(
+            `[SIMPLE_ASSET_SYNC] (${operationId}): Fetching Knowledge Asset from the network for the ` +
+                `Blockchain: ${blockchain}, Contract: ${contract}, Token ID: ${tokenId}, ` +
+                `Keyword: ${keyword}, Hash function ID: ${hashFunctionId}, Epoch: ${epoch}, ` +
+                `State Index: ${stateIndex}, Network Get Operation ID: ${getOperationId}`,
+        );
+
+        await Promise.all([
+            this.operationIdService.updateOperationIdStatus(
+                getOperationId,
+                blockchain,
+                OPERATION_ID_STATUS.GET.GET_INIT_START,
+            ),
+            this.repositoryModuleManager.createOperationRecord(
+                this.getService.getOperationName(),
+                getOperationId,
+                OPERATION_STATUS.IN_PROGRESS,
+            ),
+        ]);
+
+        try {
+            await this.commandExecutor.add({
+                name: 'networkGetCommand',
+                sequence: [],
+                delay: 0,
+                data: {
+                    getOperationId,
+                    id: ual,
+                    blockchain,
+                    contract,
+                    tokenId,
+                    state: assertionId,
+                    hashFunctionId,
+                    assertionId,
+                    stateIndex,
+                },
+                transactional: false,
+            });
+
+            await this.operationIdService.updateOperationIdStatus(
+                getOperationId,
+                blockchain,
+                OPERATION_ID_STATUS.GET.GET_INIT_END,
+            );
+
+            let attempt = 0;
+            let getResult;
+            do {
+                // eslint-disable-next-line no-await-in-loop
+                await setTimeout(SIMPLE_ASSET_SYNC_PARAMETERS.GET_RESULT_POLLING_INTERVAL_MILLIS);
+
+                // eslint-disable-next-line no-await-in-loop
+                getResult = await this.operationIdService.getOperationIdRecord(getOperationId);
```

Comment:
you forgot to update get service to store the asset ? You can check the other asset-sync branch to see how it was implemented

Reply:
#3010